node/node.go

```diff
@@ -851,15 +946,23 @@ func (n *Node) runManager(ctx context.Context, securityConfig *ca.SecurityConfig
 	if err != nil {
 		return false, err
 	}
+	// The done channel is used to signal that the manager has exited.
 	done := make(chan struct{})
+	// runErr is an error closed on by goroutine that runs the manager
 	var runErr error
+	// The context used to start this might have a logger associated with it
+	// that we'd like to reuse, but we don't wnat to use that context, so we
+	// pass to the goroutine only the logger, and create a new context with
+	//that logger.
 	go func(logger *logrus.Entry) {
 		if err := m.Run(log.WithLogger(context.Background(), logger)); err != nil {
 			runErr = err
 		}
 		close(done)
 	}(log.G(ctx))
 
+	// clearData is created here so that it can be set in the select below and
```

Comment:
Stylistic nitpick:  Since `clearData` is right by the `defer` function, it may be redundant to state that it must be set before the defer function.  However, explaining what `clearData` does, as you do below where you explain why a manager stops, might be useful (e.g. "used to indicate whether the raft data and keys should be cleared when the manager stops"), since that explanation will go nicely with the variable itself.

Totally cool with repeating said info down below where you explain the manager stopping, but commenting on the purpose of the variable where it's declared is pretty helpful too.  :)